spring-pulsar-docs/src/main/asciidoc/pulsar.adoc

```diff
@@ -142,6 +142,8 @@ You can disable the autoconfigured producer factory by providing your own bean d
 Producer<T> createProducer(String topic, Schema<T> schema) throws PulsarClientException;
 
 Producer<T> createProducer(String topic, Schema<T> schema, MessageRouter messageRouter) throws PulsarClientException;
+
+Producer<T> createProducer(String topic, Schema<T> schema, MessageRouter messageRouter, List<ProducerInterceptor> producerInterceptors) throws PulsarClientException;
```

Comment:
The latest docs commit removes many of the hardcoded areas where we specifying the APIs like in PPF. You can simply remove this line and the merge should take care of the rest (this section has been removed).